docs/source/conf.py

```diff
@@ -176,7 +176,13 @@
 # html_use_smartypants = True
 
 # Custom sidebar templates, maps document names to template names.
-# html_sidebars = {}
+html_sidebars = {
+    '**': [
+        'about.html',
+        'navigation.html',
+        'searchbox.html',
```

Comment:
I think we should add `relations.html` to the list, `donate.html` seems superfluous.